app/views/publishers/vacancies/job_applications/_banner.html.slim

```diff
@@ -42,9 +42,10 @@
           .govuk-heading-l = job_applications.unsuccessful.count
           h3.govuk-caption-s = t(".rejected", count: job_applications.unsuccessful.count)
 
-- if vacancy.enable_job_applications? && vacancy.published? && !vacancy.pending?
-  .govuk-grid-row
-    .govuk-grid-column-full
-      = render(DashboardComponent.new) do |dashboard|
-        - dashboard.navigation_item item: (link_to t("buttons.manage_listing"), organisation_job_path(vacancy.id), class: "dashboard-component-navigation__link", "aria-current": ("page" if controller_name == "vacancies"))
+.govuk-grid-row
+  .govuk-grid-column-full
+    = render(DashboardComponent.new) do |dashboard|
+      - dashboard.navigation_item item: (link_to t("buttons.manage_listing"), organisation_job_path(vacancy.id), class: "dashboard-component-navigation__link", "aria-current": ("page" if controller_name == "vacancies"))
```

Comment:
not strictly part of this task but can you make these `govuk_link_to` (and below) so they get correct focus styles. should have done this in my my PR tbh.